src/runtime_src/core/edge/drm/zocl/sched_exec.c

```diff
@@ -789,13 +711,13 @@ configure(struct sched_cmd *cmd)
 
 	SCHED_DEBUG("Configuring scheduler\n");
 	exec->num_slots       = CQ_SIZE / cfg->slot_size;
+	write_lock(&zdev->attr_rwlock);
 	exec->num_cus         = cfg->num_cus;
+	write_unlock(&zdev->attr_rwlock);
```

Comment:
There is a small window of race? 
we released lock here, but the exec->configured has not been set yet, the other threads may be able to check exec->configured == 0 and configure it again.

Reply:
Yes. Should include exec->configured.